webmails/start.py

```diff
@@ -13,14 +13,13 @@
 env = os.environ
 
 logging.basicConfig(stream=sys.stderr, level=env.get("LOG_LEVEL", "WARNING"))
+system.set_env(['ROUNDCUBE','SNUFFLEUPAGUS'])
```

Comment:
Are multiple webmails even supported? Otherwise we could just use something like `WEBMAIL` instead.

Reply:
The idea is to have one key per usage. If we have different pieces requiring keys we generate a different key each time... the key name itself doesn't matter.